test/expected/parallel-11.out

```diff
@@ -190,33 +190,32 @@ SELECT histogram(i, 10, 100000, 5) FROM "test";
 (14 rows)
 
 -- test constraint aware append with parallel aggregation
-SET max_parallel_workers_per_gather = 1;
-EXPLAIN (costs off) SELECT count(*) FROM "test" WHERE length(version()) > 0;
-                                QUERY PLAN                                 
----------------------------------------------------------------------------
- Finalize Aggregate
-   ->  Gather
-         Workers Planned: 1
-         ->  Partial Aggregate
-               ->  Result
+:PREFIX SELECT count(*) FROM "test" WHERE length(version()) > 0;
```

Comment:
This test has #workers == #chunks can we add a test where #workers > #chunks and #workers < #chunks?